src/main/java/com/nike/cerberus/command/core/CreateLoadBalancerCommand.java

```diff
@@ -34,13 +35,29 @@
 
     public static final String COMMAND_NAME = "create-load-balancer";
 
+    public static final String LOAD_BALANCER_SSL_POLICY_OVERRIDE_LONG_ARG = "--ssl-policy-override";
```

Comment:
Why not just leave it as a -P parameter?

Reply:
-P params are currently only a thing on the update stack command.